pkg/nvcdi/lib-nvml.go

```diff
@@ -118,10 +118,23 @@ func (l *nvmllib) GetDeviceSpecsBy(identifiers ...device.Identifier) ([]specs.De
 	}
 	defer func() {
 		if r := l.nvmllib.Shutdown(); r != nvml.SUCCESS {
-			l.logger.Warningf("failed to shutdown NVML: %w", r)
+			l.logger.Warningf("failed to shutdown NVML: %v", r)
 		}
 	}()
 
+	if l.nvsandboxutilslib != nil {
+		if r := l.nvsandboxutilslib.Init(l.driverRoot); r != nvsandboxutils.SUCCESS {
+			l.logger.Warningf("Failed to init nvsandboxutils: %v; ignoring", r)
```

Comment:
Warning log on line 121 is lowercase and here is upper cased, is there a reason for the diff?

Reply:
No specific reason. I will update these for consistency once we refactor.